ui/src/pages/workflows/index.vue

```diff
@@ -1,66 +1,67 @@
 <template>
-  <div class="flex flex-col gap-3 md:flex-row">
-    <!-- filters -->
-    <div
-      class="md:w-1/6 md:border-solid md:border-r md:h-screen md:min-w-[130px]"
-      style="border-color: var(--va-background-border)"
-    >
-      <!-- reset -->
-      <div class="text-right px-3">
-        <button class="va-link text-center" @click="reset_query_params">
-          Reset
-        </button>
-      </div>
-
-      <!-- status filters -->
-      <div class="text-lg font-semibold hidden md:block">Status</div>
-      <!-- reset page when user selects a new filter -->
-
-      <va-tabs
-        v-model="query_params.status"
-        :vertical="!breakpoint_sm"
-        @update:model-value="query_params.page = 1"
-        class="md:min-h-[164px]"
+  <va-inner-loading :loading="loading">
```

Comment:
Could you please remove the va-inner-loading spinner? Since the data refreshes every 10 seconds and the fetch is quite fast (100-200 ms), the spinner briefly appears and disappears, creating a glitchy effect.

In my view, a progress indicator isn’t necessary for a page that updates every 10 seconds. However, if you feel it's needed, I would recommend using a linear progress bar with a loading animation of fixed height. It shouldn't offset page elements when it appears and disappears to avoid visual disruption.

Reply:
I'd like to make some changes to pages/workflows/index.vue specifically related to useQueryPersistence and debounce after the above review is completed.